vault/core.go

```diff
@@ -1926,7 +1926,8 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 		if err := c.setupAuditedHeadersConfig(ctx); err != nil {
 			return err
 		}
-		if err := c.setupActivityLog(ctx); err != nil {
+		var wg sync.WaitGroup
```

Comment:
No Wait call?  That's fine if intended, but I'd put a comment here to say that's deliberate, since it looks funny otherwise.

Reply:
I didn't want to change the existing behavior (even though it's probably harmless). Added a comment: 3389c4396